src/NewTools-Debugger-Tests/StDebuggerContextInteractionModelTest.class.st

```diff
@@ -216,6 +216,12 @@ StDebuggerContextInteractionModelTest >> testInteractionModelArePreservedAfterCh
 	self assert: (model hasBindingOf: 'tempVariableForTest') not
 ]
 
+{ #category : 'tests' }
+StDebuggerContextInteractionModelTest >> testNeedRequestorScope [
```

Comment:
the test name does not explain the purpose of the test.
Why `needRequestorScope`should be true after initialization of the model?